src/HTTPure.purs

```diff
@@ -96,3 +99,4 @@ import HTTPure.Server
   , serveSecure
   , serveSecure'
   )
+import HTTPure.Status (Status)
```

Comment:
Think we should re-export all the status code aliases from `HTTPure.Status` as well?

Reply:
I don't think that's possible because their names clash with functions in `HTTPure.Response`, and the response functions are much more useful :)

Reply:
Yep, fair enough, agreed.  Someone could always just import `HTTPure.Status` if they really needed them.